extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -521,7 +521,7 @@ export function sanitizeProcessEnvironment(env: Record<string, string>, ...prese
 	const keysToRemove = [
 		/^ELECTRON_.$/,
 		/^VSCODE_(?!(PORTABLE|SHELL_LOGIN|ENV_REPLACE|ENV_APPEND|ENV_PREPEND)).$/,
-		/^VSCODE_(PWSH|ZSH|BASH)_ACTIVATE$/,
+		/^VSCODE_PYTHON_(PWSH|ZSH|BASH)_ACTIVATE/,
```

Comment:
The top one is actually a negative lookahead, so the python keys would already get removed from it